### PR TITLE
Adding package changelogs

### PR DESCRIPTION
**Pull request by ycombinator**
## What does this PR do?

This PR does the following:
* [x] Create initial skeleton `changelog.yml` files for every package.
* [x] Adds an item to the PR checklist that reminds package authors to add entries to their package's `changelog.yml`.

## Related issues
- https://github.com/elastic/package-spec/issues/17

### CODE REVIEW

File: packages/mysql/changelog.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: TODO
+  changes:
+    - description: initial release
+      type: enhancement                # can be one of: enhancement, bugfix, breaking-change
+      link: TODO
```
Comment (ycombinator) [inline packages/mysql/changelog.yml:6]: Hey @mtojek, any chance you could help me track down the initial PR that introduced the `mysql` package?

File: packages/nginx/changelog.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: TODO
+  changes:
+    - description: initial release
+      type: enhancement                # can be one of: enhancement, bugfix, breaking-change
+      link: TODO
```
Comment (ycombinator) [inline packages/nginx/changelog.yml:6]: Hey @mtojek, any chance you could help me track down the initial PR that introduced the `nginx` package?

File: packages/redis/changelog.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: TODO
+  changes:
+    - description: initial release
+      type: enhancement                # can be one of: enhancement, bugfix, breaking-change
+      link: TODO
```
Comment (ycombinator) [inline packages/redis/changelog.yml:6]: Hey @mtojek, any chance you could help me track down the initial PR that introduced the `redis` package?

File: packages/mysql/changelog.yml (hunk 1 above)
Comment (mtojek) [reply packages/mysql/changelog.yml:6]: In the package-registry: https://github.com/elastic/package-registry/pulls?q=is%3Apr+is%3Aclosed+MySQL+integration 

(manual prepare: https://github.com/elastic/package-registry/pull/323)

File: packages/nginx/changelog.yml (hunk 2 above)
Comment (mtojek) [reply packages/nginx/changelog.yml:6]: In the package-registry: https://github.com/elastic/package-registry/pulls?q=is%3Apr+is%3Aclosed+nginx+integration+

I would vote for: https://github.com/elastic/package-registry/pull/346

File: packages/redis/changelog.yml (hunk 3 above)
Comment (mtojek) [reply packages/redis/changelog.yml:6]: Hm... I think you ignore my other posts. I found the original source: https://github.com/elastic/integrations/tree/397992000b86a005767c505eeb8c7f093c94f84d/dev/packages/alpha - these packages were in the package-storage and we pulled them to Integrations from there.

Comment (ycombinator) [reply packages/redis/changelog.yml:6]: Ah, thanks so much! I will use https://github.com/elastic/integrations/pull/21 as the link for the `redis`, `mysql`, `nginx`, `system`, and `aws` packages initial changelog entries.

Comment (mtojek) [reply packages/redis/changelog.yml:6]: Sorry I removed the post as I believe there might be something before, but let's stick to this: https://github.com/elastic/integrations/pull/21 :)

Comment (ycombinator) [reply packages/redis/changelog.yml]: Actually, I left `system` alone as I found an even older PR that introduced the `0.0.3` version: https://github.com/elastic/integrations/pull/8.

File: packages/nginx/changelog.yml (hunk 2 above)
Comment (ycombinator) [reply packages/nginx/changelog.yml]: Ignoring in favor of https://github.com/elastic/integrations/pull/675#discussion_r577110505.

File: packages/mysql/changelog.yml (hunk 1 above)
Comment (ycombinator) [reply packages/mysql/changelog.yml]: Ignoring in favor of https://github.com/elastic/integrations/pull/675#discussion_r577110505.

**Review by mtojek — APPROVED** [review]
If the CI is happy, I'm happy too :)

**Comment by ycombinator** [comment]
> If the CI is happy, I'm happy too :)

Actually, now that this PR is ready, I will update `elastic-package` with the changes from https://github.com/elastic/package-spec/pull/131 and then update this PR with the changes to `elastic-package`. Then let's see if CI is still happy on this PR. 🙂

**Comment by ycombinator** [comment]
I just added a commit to this PR to update the dependency on `elastic-package`: 44ffaed23cce8f16356a41081ce183465ddab591. This version of `elastic-package` will enforce `changelog.yml` files. So if CI goes green on this PR now, I will go ahead and merge it.

**Comment by ycombinator** [comment]
CI is failing on two system tests: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations/detail/PR-675/19/tests. Re-running to see if this is a flaky failure: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations/detail/PR-675/20/tests. Also re-running the `master` build so we can check if the failures are isolated to this PR: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations/detail/master/82/tests.

**Comment by ycombinator** [comment]
> CI is failing on two system tests: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations/detail/PR-675/19/tests.

> Re-running to see if this is a flaky failure: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations/detail/PR-675/20/tests.

The same two system tests failed again so this doesn't appear to be a flaky failure.

> Also re-running the `master` build so we can check if the failures are isolated to this PR: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations/detail/master/82/tests.

All tests passed on this `master` build. So the failures are definitely related to this PR.

Will investigate both system test failures in this PR tomorrow.

**Comment by mtojek** [comment]
It looks really weird what happened here. I wonder if it's not due to other changes in elastic-package.

**Comment by ycombinator** [comment]
Thanks for the fixes, @mtojek! Any idea why these issues only came up with the newer version of `elastic-package`? Just curious what changed in `elastic-package` to expose these two problems that we weren't catching before. 🤔 

And thanks for the heads up about #477. I will update this PR with `changelog.yml` files for those packages as well.

**Comment by mtojek** [comment]
I suppose that they're related with 7.12.0 bump. I don't have an idea if sth has been added to filebeat/metricbeat there.

**Comment by ycombinator** [comment]
Hmm, same test failures as before. Plus now we have 3 skipped tests too for the `radware` package. Investigating...

**Comment by mtojek** [comment]
Did you apply the suggested fixes for zoom and kubernetes? I don't see them here: https://github.com/elastic/integrations/pull/675/files

**Comment by ycombinator** [comment]
> Did you apply the suggested fixes for zoom and kubernetes? I don't see them here: https://github.com/elastic/integrations/pull/675/files

I thought I did when I rebased this PR on `master`. Let me check...

**Comment by mtojek** [comment]
Sorry for the confusion, I didn't put them on master, but in a side branch (just to investigate problems with this PR).

**Comment by ycombinator** [comment]
Ah, sorry, I missed that. I will cherry-pick those commits over to this PR now, thanks!

**Comment by mtojek** [comment]
> Ah, sorry, I missed that. I will cherry-pick those commits over to this PR now, thanks!

Most likely you will need some formatting as well.

**Comment by mtojek** [comment]
... and `elastic-package build` for Kubernetes.

**Pull request merged**